html-tables/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <title>Student Grade Table</title>
+    </head>
+    <body>
+      <h1>
+        Student Grade Table
+      </h1>
+      <table>
+        <tbody>
```

Comment:
make sure you are using the thead element for the `head` of the table where the heading are, and the tbody for the body of the table